### PR TITLE
DBZ-6801 Introduce JDBC sink option `dialect.sqlserver.identity.insert`

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-6801

Documents https://github.com/debezium/debezium-connector-jdbc/pull/27

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/jdbc.adoc [hunk 1]
```diff
@@ -701,6 +701,10 @@ The placeholder, `${topic}`, is replaced by the topic name.
 |Specifies the schema name where the PostgreSQL PostGIS extension is installed.
 The default is `public`; however, if the PostGIS extension was installed in another schema, this property should be used to specify the alternate schema name.
 
+|[[jdbc-property-dialect-sqlserver-identity-insert]]<<jdbc-property-dialect-sqlserver-identity-insert, `+dialect.sqlserver.identity.insert+`>>
+|`false`
+|Specifies whether the connector automatically enables identity inserts before an _insert_ or _upsert_ operation and is disabled immediately afterward.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/jdbc.adoc:706]:
```suggestion
|Specifies whether the connector automatically sets an `IDENTITY_INSERT` before an `INSERT` or `UPSERT` operation into the identity column of SQL Server tables , and then unsets it immediately after the operation.
```

File: documentation/modules/ROOT/pages/connectors/jdbc.adoc [hunk 2]
```diff
@@ -701,6 +701,10 @@ The placeholder, `${topic}`, is replaced by the topic name.
...
+The default is `false`, meaning an insert or upsert into a SQL Server table that has an `IDENTITY` column throws a SQL exception.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/jdbc.adoc:707]:
```suggestion
When the default setting (`false`) is in effect, an `INSERT` or `UPSERT` operation into the `IDENTITY` column of a table results in a SQL exception.
```

File: documentation/modules/ROOT/pages/connectors/jdbc.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/jdbc.adoc:706]: The convention in the SQL Server documentation seems to be to capitalize SQL commands, so I followed that model.

**Review by roldanbob — CHANGES_REQUESTED** [review]

**Comment by Naros** [comment]
Applied to main (=2.4)

**Pull request merged**